UIDokan/Sources/UIDokan/Tools/TableViewHelper.swift

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
Let rename `UITableViewHelper.swift` to `UITableView+DequeueHelper.swift` and move it to `Extensions`